src/modules/visits/types.ts

```diff
@@ -76,6 +82,7 @@ export interface Visit {
 	spaceServiceDescription?: string;
 	startAt: string;
 	status: VisitStatus;
+	accessType: string;
```

Comment:
Should we use the enum type here everywhere instead of strings?
```suggestion
	accessType: VisitAccessType;
```